string name                         # object's name
geometry_msgs/Vector3 scale         # new scale
--- 
